app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -47,7 +51,26 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               className="w-full flex-grow-0"
               label="Description"
               name="description"
-              rows={8}
+              rows={1}
+            />
+            <Select
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Assigned Contact"
+              name="contact"
+              options={ASSIGNED_CONTACT}
+              placeholder="Select Role"
+            />
+            <Select
+              isMulti
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Tags"
+              name="tag"
+              options={TAGS}
+              placeholder="Select Role"
```

Comment:
The label uses the keyword `Tag`. But the placeholder uses the keyword `Role`. This doesn't make any sense to me. The end user/consumer will be confused seeing this in the UI. Pay close attention to what you are displaying in the UI. The whole purpose of creating Web apps is to consumer's life easier.